Snowdog_Components/components/Templates/product-view/_product-view.scss

```diff
@@ -327,9 +327,6 @@
     &__grouped-table {
         margin: $product-view__grouped-table-margin;
         border-width: $product-view__grouped-table-border-width;
-        @include mq($screen-m) {
-            border-width: $product-view__grouped-table-border-width\@medium;
-        }
```

Comment:
please remove this variable from `Snowdog_Components/components/Templates/product-view/_product-view-variables.scss`

Reply:
and please add note in CHANGELOG.md